src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -70,8 +71,16 @@ public class JwtSecurityTokenHandler : SecurityTokenHandler
             { SecurityAlgorithms.RsaSha256Signature, SecurityAlgorithms.RsaSha256 },
             { SecurityAlgorithms.RsaSha384Signature, SecurityAlgorithms.RsaSha384 },
             { SecurityAlgorithms.RsaSha512Signature, SecurityAlgorithms.RsaSha512 },
+            { SecurityAlgorithms.RsaOaepKeyWrap, GetRsaOaepMapping() },
```

Comment:
This issue is also existing in JsonWebTokenHandler.
If we make the switch in the ctor of X509EncryptingCredentials we will modify both cases.

Reply:
Move the changes to the X509EncryptingCredentials and did tests for both handlers. Thanks Brent!